src/utils/jsonwebtoken.js

```diff
@@ -0,0 +1,124 @@
+// packages
+const jwt = require('jsonwebtoken');
+const dotenv = require('dotenv').config();
+const crypto = require('crypto');
+const fs = require('fs');
+const path = require('path');
+
+/** 
+ * @param {object} userInfo - validated user id
+ * @param {string} expirationTime - token validity
+ */
+
+/**Basic / symmetric jwt generation */
+const generatingToken = function (userInfo, expirationTime) {
+    // sample of generated payload
+    const {userId, fullName, email} = userInfo;
+    const payload = {
+        sub: userId,
+        name: fullName,
+        email,
+        iat: Date.now(),
```

Comment:
I think the jwt signing function itself provides iat and exp...so you might not really need this here.